### ISSUE TITLE
Unable to resolve module `AccessibilityInfo`

### ISSUE THREAD

**Issue by kosiakMD** [Vizir/react-native-autocomplete-select#4]
RN error: Unable to resolve module `AccessibilityInfo` after installing your module

**Comment by alainib** [comment]
get the same error.
how did you resolved it ?

**Comment by kosiakMD** [comment]
@alainib, didn't decide, took another package :(

**Comment by LaloHao** [comment]
Delete the react-native folder that got installed as a dependency from this package `rm -r ./node_modules/react-native-autocomplete-select/node_modules/react-native` then start the server resetting the cache `npm start -- --reset-cache`
